Extensions/Today/TodayViewController.swift

```diff
@@ -32,25 +34,24 @@ private struct TodayUX {
 
 @objc (TodayViewController)
 class TodayViewController: UIViewController, NCWidgetProviding {
-
     var copiedURL: URL?
 
     fileprivate lazy var newTabButton: ImageButtonWithLabel = {
         let imageButton = ImageButtonWithLabel()
         imageButton.addTarget(self, action: #selector(onPressNewTab), forControlEvents: .touchUpInside)
         imageButton.label.text = TodayStrings.NewTabButtonLabel
-
         let button = imageButton.button
-        button.frame = CGRect(width: 60.0, height: 60.0)
-        button.backgroundColor = UIColor.white
-        button.layer.cornerRadius = button.frame.size.width/2
-        button.clipsToBounds = true
-        button.setImage(UIImage(named: "search"), for: .normal)
+        button.setImage(UIImage(named: "search-button")?.withRenderingMode(.alwaysOriginal), for: .normal)
+        button.imageView?.contentMode = .scaleAspectFit
         let label = imageButton.label
-        label.tintColor = UIColor(named: "widgetLabelColors")
-        label.textColor = UIColor(named: "widgetLabelColors")
+        if #available(iOS 13, *) {
```

Comment:
You could potentially get rid of the color set and just use the TodayUX.labelColor
then you won't have to do the #available iOS 13 part. 
Ex. 

```
        let label = imageButton.label
        label.textColor = TodayUX.labelColor
        label.tintColor = TodayUX.labelColor
``

Reply:
the color set is there to handle the dark/light mode support. I was thinking I can make a theme object and use it, if it's accessible on this target.

Reply:
You are right, I missed the dark mode part. Theme object sounds good but lets park our thoughts and come back to it during our refactor.